run

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+export PATH=./node_modules/.bin:$PATH
+
+function build(){
+    rm -fr dist || true
+    node scripts/build.cjs
+}
+
+function build-website(){
+    mkdir -p dist/website
+    cp website/index.html dist/website/index.html
```

Comment:
This is the entry point for what will probably end up being a rollup script runs to bundle up the docs.